src/appkit.cs

```diff
@@ -9940,20 +10130,42 @@ interface NSLayoutAnchor<AnchorType> : NSCoding, NSCopying
 
 		[Export ("constraintLessThanOrEqualToAnchor:constant:")]
 		NSLayoutConstraint ConstraintLessThanOrEqualToAnchor (NSLayoutAnchor<AnchorType> anchor, nfloat constant);
+
+		[Mac (10, 12)]
+		[Export ("name")]
+		string Name { get; }
+
+		[Mac (10, 12)]
+		[NullAllowed, Export ("item", ArgumentSemantic.Weak)]
+		NSObject Item { get; }
+
+		[Mac (10, 12)]
+		[Export ("hasAmbiguousLayout")]
+		bool HasAmbiguousLayout { get; }
+
+		[Mac (10, 12)]
+		[Export ("constraintsAffectingLayout")]
+		NSLayoutConstraint[] ConstraintsAffectingLayout { get; }
 	}
 
 	[Mac (10,11)]
 	[BaseType (typeof(NSLayoutAnchor<NSLayoutXAxisAnchor>))]
 	[DisableDefaultCtor] // Handle is nil
 	interface NSLayoutXAxisAnchor
 	{
+		[Mac (10,12)]
+		[Export ("anchorWithOffsetToAnchor:")]
+		NSLayoutDimension AnchorWithOffset (NSLayoutXAxisAnchor otherAnchor);
```

Comment:
I'm not sure I like this one. WithOffset describes that it's offset of this other anchor.